### ISSUE TITLE
[Bug]: > 0.16 working with old firmware?

### ISSUE THREAD

**Issue by danrichardson** [Patrick762/hassio-bluetti-bt#67]
### What happened?

I can't get v 0.17+ to run on my AC300 with older firmware.  v 0.16 works great.  I'm about to send this unit back (because it won't update to the latest firmware - Wifi is unstable).

My firmware(s) are:
IoT: v9014.09 ARM: v4032.10 DSP: v4031.16 Battery: v1024.04

### Version

0.0.17

### What device are you seeing the problem on?

AC300

### What bluetooth connection are you using?

ESPHome bluetooth proxy

### Have you changed the integration settings?

false

### Relevant log output

```shell
No log output - just constantly says it can't connect.  The front panel of the AC300 says there's a Bluetooth connection.
```

**Comment by Patrick762** [comment]
@danrichardson Should be solved in version 0.0.20

**Comment by danrichardson** [comment]
Confirm working with 0.0.21.  Thanks!
